tests/e2e/lib/ContainerFile.template

```diff
@@ -69,6 +69,7 @@ WORKDIR /root
 
 RUN gem install ruby-dbus
 RUN dnf -y copr enable @centos-automotive-sig/bluechi-snapshot centos-stream-9
+RUN dnf -y copr enable @centos-automotive-sig/qm-snapshot centos-stream-9
```

Comment:
I think it is added by container script, this way, for Packit it will never use PR build rpm